types/koa/index.d.ts

```diff
@@ -724,6 +718,12 @@ declare namespace Application {
     interface Context extends ParameterizedContext {}
 
     type Next = () => Promise<any>;
+
+    /**
+     * Make HttpError available to consumers of the library so that consumers don't
+     * have a direct dependency upon `http-errors`
```

Comment:
This comment seems applicable to the PR, but not so much to the codebase.

would it be correct in saying that a `HttpError` is thrown when using `ctx.assert` or `ctx.throw`? I think a comment for this should reflect why/how one might need to access this type

Reply:
I just copied the exact comment from the Koa code base, but I agree.

I changed the comment.